modules/nw-ingress-controller-configuration-proxy-protocol.adoc

```diff
@@ -10,9 +10,14 @@ A cluster administrator can configure https://www.haproxy.org/download/1.8/doc/p
 
 This feature is not supported in cloud deployments. This restriction is because when {product-title} runs in a cloud platform, and an IngressController specifies that a service load balancer should be used, the Ingress Operator configures the load balancer service and enables the PROXY protocol based on the platform requirement for preserving source addresses.
 
+[IMPORTANT]
+====
+You must configure both {product-title} and the external load balancer to either use the PROXY protocol, or to use TCP.
+====
+
 [WARNING]
 ====
-To prevent connection failure, configure both the Ingress Controller and the load balancer to use the PROXY protocol.
+The PROXY protocol is unsupported for the default Ingress Controller with IPI clusters on non-cloud platforms that use a Keepalived Ingress VIP.
```

Comment:
Do not use [IPI](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#installer-provisioned-infrastructure).

Should "a Keepalived Ingress VIP" be closer to "the default Ingress Controller" in your clause stackup?

Reply:
@kalexand-rh I'm not sure what you mean. iiuc, then no, because that feature about the IngressVIP is platform-specific so I want it to be more closely tied to that part of the clause, and less tied to the part about the Ingress Controller. But maybe this sentence isn't all that clear...what about:

`For installer-provisioned clusters on non-cloud clusters that use a Keepalived Ingress VIP, the PROXY protocol is unsupported for the default Ingress Controller.`

Reply:
With no context, it sounded like the Ingress VIP and Ingress Controller should be more closely related, but if they're not, then either one will work!